nameable.rb

```diff
@@ -0,0 +1,30 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
+
+class Decorator < Nameable
+  def initialize(nameable)
+    @nameable = nameable
```

Comment:
- [ ] I will encourage you to access the nameable once you've changed them to different file something like `attr_accessor :nameable`